LayoutTests/accessibility/color-well-legacy.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN"><!-- webkit-test-runner [ InputTypeColorEnhancementsEnabled=false ] -->
+<html>
+<head>
+<script src="../resources/js-test.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+</head>
+<body>
+
+<input id="empty_colorwell" type="color">
+<input id="good_colorwell" type="color" value="#ff0000">
+<input id="bad_colorwell" type="color" value="purple">
+
+<script>
+    description("This test checks the role of ColorWellRole on an input with type=color");
+
+    if (window.accessibilityController) {
+        let axColorwell = accessibilityController.accessibleElementById("empty_colorwell");
+        debug("Role of input type=color is: " + axColorwell.role);
```

Comment:
In newer accessibility tests, we don't use `debug` or `description`, as they write to the DOM immediately, which has made a noticeable difference in test speed (though less so in smaller tests like this). Instead, we would declare:

```
var output = "This test checks the role of ColorWellRole on an input with type=color\n\n";
```

Where `description` is, and replace `debug` with `output += "foo\n";`

And then `debug(output)` at the end.

Here's an example of a test following this format:

https://github.com/WebKit/WebKit/blob/main/LayoutTests/accessibility/changing-aria-hidden-with-display-none-parent.html#L16